app/src/main/res/layout/view_simple_video.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  android:id="@+id/simpleVideoLayout"
+<RelativeLayout android:id="@+id/simpleVideoLayout"
```

Comment:
Place `android:id` after the XML namespace attribute?

Reply:
Auto format does this. Since in other xmls in the project, we are following the same formatting, I am going to keep it this way for now.
Maybe we can update our formatter.